scap/scc/scan.sls

```diff
@@ -9,6 +9,15 @@ create scc output directory:
     - name: '{{ scc.output_dir }}'
     - makedirs: True
 
+set permissions option:
```

Comment:
I think this makes more sense in the scc/init.sls instead, which handles the install/config, rather than the scan.

Reply:
Agreed.  I'll move it to scc/init.sls.